Application/EdFi.Ods.Standard/Standard/5.0.0/Artifacts/Metadata/DatabaseViews.generated.json

```diff
@@ -1,278 +1,278 @@
-[
```

Comment:
This is a generated file. Do we need to have it checked in the repo?

Reply:
If I remember correctly, checking it in the repo was started fairly recently and was a deliberate choice, but I don't remember the reason behind it